spec/lib/packaging/util/git_tag_spec.rb

```diff
@@ -11,26 +11,26 @@
   context "sha?" do
     it "sets ref type as a sha when passed a sha" do
       git_tag = Pkg::Util::Git_tag.new("https://github.com/puppetlabs/leatherman.git", "4eef05389ebf418b62af17406c7f9f13fa51f975")
-      expect(git_tag.sha?).to eq(true)
+      expect(git_tag.sha?).to be(true)
```

Comment:
make these `be true` to match spec/lib/packaging/util/execution_spec.rb as this is the only file that doesn't match.